trl/trainer/ppo_trainer.py

```diff
@@ -461,15 +461,21 @@ def step(
                 model_inputs["decoder_attention_mask"] = self.accelerator.pad_across_processes(
                     model_inputs["decoder_attention_mask"], dim=1, pad_index=0, pad_first=pad_first
                 )
+            else:
+                model_inputs['labels'] = self.accelerator.pad_across_processes(
```

Comment:
`labels` were used for decoder based models @lvwerra and I mistakenly deleted then in the previous PR